discovery-infra/download_logs.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import os
+from argparse import ArgumentParser
+
+from logger import log
+from utils import recreate_folder
+from install_cluster import download_logs_from_all_hosts
+from assisted_service_api import create_client, InventoryClient
+
+FAILED_STATUSES = ["error"]
+DEFAULT_DEST = 'build'
+
+def main():
+    args = handle_arguments()
+    client = create_client(url=args.inventory_url)
+
+    if args.cluster_id:
+        donwload_logs(client, args.cluster_id, args.dest, args.force)
+    else:
+        clusters = client.clusters_list()
+
+        if not clusters:
+            log.info('No clusters were found')
+            return
+
+        for cluster in clusters:
+            if cluster['status'] in FAILED_STATUSES:
+                donwload_logs(client, cluster['id'], args.dest, args.force)
+
+
+def donwload_logs(client: InventoryClient, cluster_id: str, dest: str, force: bool):
```

Comment:
really? types? for what? all the code we don't have and here you add it? It will not allow to run same code on older python version and many times we need due to old OS